scripts/src/main/resources/scripts/command/vscode

```diff
@@ -180,6 +205,11 @@ do
     shift
     doAddPlugin "${@}"
     exit ${?}
+  elif [ "${1}" = "clean-setup" ]
+  then
+    doCleanSetup "${2}"
+    doStartVsCode
```

Comment:
I added this feature while I was doing the integration-test for `vscode`. I had to do a workaround there, because the `devonfw-extension-package` broke the whole workflow. So I thought it would be nice to have an option to install vscode without plugins in "clean" way.

I am not sure, but probably user with poor internet connection will also have the problem that the plugin takes a lot of time to install. There is already an issue for the extension pack https://github.com/devonfw/extension-pack-vscode/issues/12. If there will be an update, the clean installation is not so relevant anymore or we disable the extension plugin as default.

Reply:
> Maybe it is better to add an option like -no-plugins instead, if we need this at all.

If you still think it would be good to offer a way to skip plugin installation, that was my recommendation.
Otherwise we can entirely remove the "clean-setup".

I am also happy to have the devonfw extension pack as recommendation by default instead of a fully automated installation for every user by force as https://github.com/devonfw/extension-pack-vscode/issues/12 does not seem to make any progress.
So once we have this PR completed and merged, my understanding is that VSCode plugins with `plugin_active=false` will not be installed but added as "recommendation". So after merge, we can then change our settings accordingly:
https://github.com/devonfw/ide-settings/blob/d98a888be2e02922cc5e81f7c186e9c8db40f89f/vscode/plugins/devonfw-extension-pack.properties